### ISSUE TITLE
Card Index - very different from others

### ISSUE THREAD

**Issue by Michele0512** [hfg-gmuend/openmoji#361]
Card index in openmoji is very different from other emoji sets. Moreover it is very similar to card index divider and it can be very easily confused with this.

In others sets of emojis, card index is usually as a rotating device with a contact card showing a picture and some text

Please review and consider changing it to make consistent with other sets

**Comment by b-g** [comment]
Many thanks for the feedback! But please be more verbose and provide images, samples, sketches ... it is very hard to discuss these things "text-only" :)

**Comment by Michele0512** [comment]
It is very obvious checking emojipedia where there are emojis from all differen sets including openmoji.

Card index is usually represented by a rotating device with multiple cards, one of these cards is visible and showing (usually) a picture with some text (ie contact information for that person).

Card index is represented in openmoj almost the same as card index dividers! It almost seems to be a different/variant image of dividers rather than a card index!

**Comment by Michele0512** [comment]
![card index](https://user-images.githubusercontent.com/56539481/141294999-14de1bcf-db54-49d6-8ef4-61545994093b.jpg)

**Comment by zerorabb** [comment]
I agree the two OpenMoji emojis look virtually indistinguishable. I think the Card Index one should definitely change to be more like the others and depict an actual card index. 
Maybe the Card Index Dividers one should be yellow for clarity too?
